sdk/documenttranslation/azure-ai-documenttranslation/tests/test_translation.py

```diff
@@ -0,0 +1,217 @@
+        ]
+
+        # submit job
+        job_detail = client.create_translation_job(translation_inputs)
+        self.assertIsNotNone(job_detail.id)
+
+        # wait for result
+        job_result = client.wait_until_done(job_detail.id)
+
+        # assert
+        self.assertEqual(job_result.status, "Succeeded")  # job succeeded
+        self.assertIsNotNone(job_result.created_on)
+        self.assertIsNotNone(job_result.last_updated_on)
+        self.assertIsNotNone(job_result.documents_total_count)
+        self.assertIsNotNone(job_result.documents_failed_count)
+        self.assertIsNotNone(job_result.documents_succeeded_count)
+        self.assertIsNotNone(job_result.documents_in_progress_count)
+        self.assertIsNotNone(job_result.documents_not_yet_started_count)
+        self.assertIsNotNone(job_result.documents_cancelled_count)
+        self.assertIsNotNone(job_result.total_characters_charged)
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    def test_multiple_sources_single_target(self, client):
+        self._setup()  # set up test resources
+
+        # prepare translation inputs
+        translation_inputs = [
+            DocumentTranslationInput(
+                source_url=self.source_container_sas_url,
+                targets=[
+                    TranslationTarget(
+                        target_url=self.target_container_sas_url,
+                        language_code="es"
+                    )
+                ]
+            ),
+            DocumentTranslationInput(
+                source_url=self.additional_source_container_sas_url,
+                targets=[
+                    TranslationTarget(
+                        target_url=self.additional_target_container_sas_url,
+                        language_code="fr"
+                    )
+                ]
+            )
+        ]
+
+        # submit job
+        job_detail = client.create_translation_job(translation_inputs)
+        self.assertIsNotNone(job_detail.id)
+
+        # wait for result
+        job_result = client.wait_until_done(job_detail.id)
+
+        # assert
+        self.assertEqual(job_result.status, "Succeeded")  # job succeeded
+        self.assertIsNotNone(job_result.created_on)
+        self.assertIsNotNone(job_result.last_updated_on)
+        self.assertIsNotNone(job_result.documents_total_count)
+        self.assertIsNotNone(job_result.documents_failed_count)
+        self.assertIsNotNone(job_result.documents_succeeded_count)
+        self.assertIsNotNone(job_result.documents_in_progress_count)
+        self.assertIsNotNone(job_result.documents_not_yet_started_count)
+        self.assertIsNotNone(job_result.documents_cancelled_count)
+        self.assertIsNotNone(job_result.total_characters_charged)
+
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    def test_single_source_single_target_with_prefix(self, client):
+        self._setup()  # set up test resources
```

Comment:
the prefix passed in the DocumentTranslationInput will apply to the documents in the container. we prefix the container with "src", but we need to explicitly say to prefix the document so the setup code names it correctly:

```suggestion
        self._setup(blob_prefix="src")  # set up test resources
```

Reply:
aha ok!
i was more focused on writing the tests, didn't look thoroughly into this setup function
i think i'll modify it a little to give the test more control on managing containers and files